src/composables/global.ts

```diff
@@ -31,21 +36,39 @@ export function globalStore() {
         }, 40000)
     }
 
-    //Notifications
+    /**
+     * @notice Toggles Notification Layout.
+     */
     const toggleNotificationOpen = () => {
         globalState.isNotificationOpen = !globalState.isNotificationOpen
     }
 
+    /**
+     * @notice Delete notification from Layout.
+     */
     const deleteNotification = (index: number) => {
         globalState.notifications.splice(index, 1)
     }
 
+    /**
+     * @notice Delete notification from global view and put it to Layout.
+     */
     const deleteNewNotification = (index: number) => {
         clearTimeout(newNotificationTimer)
         globalState.notifications.push(globalState.newNotifications[index])
         globalState.newNotifications.splice(index, 1)
     }
 
+    /**
+     * @notice Add global notifications.
+     *
+     * @param payload: Object Notification Data
+     * @param payload.style: Number  0 - information style, 1 - success style, 2 - error style
+     * @param payload.link: String  Third-party link
+     * @param payload.title String  Notification Title
+     * @param payload.content String  Notification text content
+     * @param saveGlobally: Boolean Saved notification to side bar by default
+     */
```

Comment:
we will need to separate this into its own component, as well to have a `readme.md` documenting how to use it very much like the charts, with further informational updates